docs/contribute/development.rst

```diff
@@ -0,0 +1,65 @@
+# How to set up Polaris development environment
+
+If you would like to contribute to Polaris, this document will guide you to set up your development environment on
+Ubuntu systems.
+
+## Ubuntu 20.04LTS
```

Comment:
Were you planning to add different instructions for Ubuntu 18? Or is it the same? If its the same flow I would remove this header.

Reply:
Yes, I will add instructions for installing pip, pipenv, pulling django-polaris to the document. I will find a way to add docker-compose instruction.

Reply:
I will remove the header. I don't think it makes sense to use Ubuntu 18 when we have 20.04LTS